src/component/archive/ArchiveTable/ReviveArchivedFeatureCell/ReviveArchivedFeatureCell.tsx

```diff
@@ -0,0 +1,27 @@
+import { VFC } from 'react';
+import { ActionCell } from '../../../common/Table/cells/ActionCell/ActionCell';
+import { Undo } from '@mui/icons-material';
+import PermissionIconButton from '../../../common/PermissionIconButton/PermissionIconButton';
+import { ADMIN } from '../../../providers/AccessProvider/permissions';
+
+interface IReviveArchivedFeatureCell {
+    onRevive: (featureName: string) => Promise<void>;
+    project: string;
+}
+
+export const ReviveArchivedFeatureCell: VFC<IReviveArchivedFeatureCell> = ({
+    onRevive,
+    project,
+}) => {
+    return (
+        <ActionCell>
+            <PermissionIconButton
+                onClick={onRevive}
+                projectId={project}
+                permission={ADMIN}
```

Comment:
It's not obvious, but the correct permission here is `UPDATE_FEATURE`

Reply:
We should also add: `tooltipProps={{ title: 'Revive feature' }}` for accessability here.